bundles/jsch-agentproxy-jsch/pom.xml

```diff
@@ -0,0 +1,94 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!--
+Licensed to the Apache Software Foundation (ASF) under one or more
+contributor license agreements.  See the NOTICE file distributed with
+this work for additional information regarding copyright ownership.
+The ASF licenses this file to You under the Apache License, Version 2.0
+(the "License"); you may not use this file except in compliance with
+the License.  You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+-->
+
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+  <parent>
+    <artifactId>jclouds-karaf</artifactId>
+    <groupId>org.apache.jclouds</groupId>
+    <version>2.0.0-SNAPSHOT</version>
+    <relativePath>../..</relativePath>
+  </parent>
+  <modelVersion>4.0.0</modelVersion>
+
+  <groupId>org.apache.jclouds.karaf.bundles</groupId>
+  <artifactId>jsch-agentproxy-jsch</artifactId>
+  <name>jclouds :: Karaf :: JSCH Agentproxy Shaded Bundle</name>
+  <packaging>bundle</packaging>
+
+  <dependencies>
+    <dependency>
+      <groupId>org.apache.servicemix.bundles</groupId>
+      <artifactId>org.apache.servicemix.bundles.jsch-agentproxy-jsch</artifactId>
+      <version>${jsch.agentproxy.bundle.version}</version>
+    </dependency>
+  </dependencies>
+
+  <properties>
+    <import.packages>
```

Comment:
Just to ensure I'm understanding what's going on: with this property, we're effectively "overriding" the import package spec from the original bundle and packaging up all the code with a manifest resulting from this?